Source/WebInspectorUI/UserInterface/Views/QuickConsole.js

```diff
@@ -467,3 +467,5 @@ WI.QuickConsole = class QuickConsole extends WI.View
         this.element.classList.toggle("showing-log", WI.isShowingConsoleTab() || WI.isShowingSplitConsole());
     }
 };
+
+WI.QuickConsole.MinimumHeight = 30; /* Keep in sync with `--console-prompt-min-height in Variables.css` */
```

Comment:
NIT: the "in Variables.css"` part isn't really necessary as there should only be one definition of this CSS variable (i.e. easily discoverable by searching the code)